src/main/resources/db/migration/V1_0__create_schema.sql

```diff
@@ -0,0 +1,22 @@
+create table roles
+(
+  id serial not null,
+  role varchar(20) not null unique default '',
+  primary key (id)
+);
+
+create table users
+(
+  id serial not null,
+  email varchar(50) not null unique,
```

Comment:
You have method - **UserEntity getUserByEmail**. But don't have index on this field

Reply:
done